src/styles/style.scss

```diff
@@ -24,23 +27,216 @@
 ******************************************************/
 
 .cards {
-  display: grid;
-  margin: 0 auto;
-  max-width: 55.375rem;
-  padding-top: 1rem;
-  margin-bottom: 5.7813rem;
-  width: 100%;
+    display: grid;
+    margin: 0 auto;
+    max-width: 55.375rem;
+    padding-top: 1rem;
+    margin-bottom: 5.7813rem;
+    width: 100%;
 
   @include media('tablet') {
-    gap: 1.5rem;
+      gap: 1.5rem;
     grid-template-columns: repeat(2, 1fr);
     margin-bottom: 2.9375rem;
   }
 
   @include media('desktop') {
-    gap: 3.75rem;
+      gap: 3.75rem;
     max-width: 67.5rem;
     margin-bottom: 4.66rem;
   }
+  
+}
+
+//--------------------UNDERSTANDING NF1--------------------
+//nf1-section
+ .nf1-margin{
+    margin-bottom: 0rem!important;
+    @media screen and (min-width: 768px) {
+        margin-bottom: 16rem!important;
+    }
+} 
+
+.nf1{
+    padding-left: 34px;
```

Comment:
![image](https://user-images.githubusercontent.com/31933833/228303743-68e78506-647f-46e7-83e1-309519294d76.png)

This padding must be deleted